packages/tools/src/synchronizers/callbacks/slabThicknessSyncCallback.ts

```diff
@@ -0,0 +1,29 @@
+import { getRenderingEngine, Types } from '@cornerstonejs/core';
+
+/**
+ * Synchronizer callback to synchronize the slab thickness.
+ */
+export default function slabThicknessSyncCallback(
+  _synchronizerInstance,
+  sourceViewport: Types.IViewportId,
+  targetViewport: Types.IViewportId
+): void {
+  const renderingEngine = getRenderingEngine(targetViewport.renderingEngineId);
+  if (!renderingEngine) {
+    throw new Error(
+      `No RenderingEngine for Id: ${targetViewport.renderingEngineId}`
+    );
+  }
+
+  const tViewport = renderingEngine.getViewport(targetViewport.viewportId);
+  const sViewport = renderingEngine.getViewport(sourceViewport.viewportId);
+
+  const slabThickness = (
+    sViewport as Types.IVolumeViewport
+  ).getSlabThickness?.();
+  if (!slabThickness) {
+    return;
+  }
+  (tViewport as Types.IVolumeViewport).setSlabThickness?.(slabThickness);
+  tViewport.render();
+}
```

Comment:
didn't we decide to do a properties synchronizer?

Reply:
I'm working on a view position synchronizer, but I think the slab thickness stands on it's own as something that one would turn on/off by itself.